### PR TITLE
task(settings): Add new icons for 2FA flows

### PR DESCRIPTION
**Pull request by dschom**
## Because

- We want to get icons ready to go!

## This pull request

- Adds icons (in new icons folder)
- Moves things are icons to this folder from images folder
- Moves Prepared Image into its own component for easy reuse
- Updates storybooks

## Issue that this pull request solves

Closes: FXA-10203

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

See storybooks components/Icons

## Other information (Optional)

Note the ticket also calls for an icon that looks like this light bulb, but I don't see that in our figma designs atm, so this was not included.

### CODE REVIEW

File: packages/fxa-settings/src/components/Icons/index.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  />
+);
+
+export const CheckmarkCircleFullGreenIcon = ({
+  className,
+  ariaHidden,
+}: ImageProps) => (
+  <PreparedIcon
+    Image={CheckmarkCircleFullGreen}
+    ariaLabel="Check"
```
Comment (vpomerleau) [inline packages/fxa-settings/src/components/Icons/index.tsx:100]: This also applies to the other checkmark icons - 'success' or 'enabled' might be more semantically correct than 'check' for a screen-reader user, but which one to use depends on context (in success banner for example vs in settings rows). Maybe we'd need a prop to allow selection of either aria-label 🤔

File: packages/fxa-settings/src/components/Icons/index.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    ariaLabel="Canadian Flag"
+    ariaLabelFtlId="canadian-flag-icon-aria-label"
+    Image={FlagCanada}
+    {...{ className, ariaHidden }}
+  />
+);
+
+export const FlagUsaIcon = ({ className, ariaHidden }: ImageProps) => (
+  <PreparedIcon
+    ariaLabel="USA Flag"
```
Comment (vpomerleau) [inline packages/fxa-settings/src/components/Icons/index.tsx:137]: Maybe "United-States Flag"?

**Review by vpomerleau — APPROVED** [review]
LGTM! Thank you for splitting out Icons from Images, that makes sense as our collection grows.

I just have one comment about aria-labels.

FYI - Lauren also has a PR for images so might be good to check in with her before merging to minimize merge conflicts for either of you :)

File: packages/fxa-settings/src/components/Icons/en.ftl [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+code-icon-aria-label =
+    .aria-label = Code
+
+# Used to select Canada as country code for phone number
+canadian-flag-icon-aria-label =
+    .aria-label = Canadian Flag
+
+# Used to select United States as a country code for phone number
+usa-flag-icon-aria-label =
+    .aria-label = USA Flag
```
Comment (bcolsson) [inline packages/fxa-settings/src/components/Icons/en.ftl:39]: I'd recommend `United States` here to be consistent with what we use for the region name in this and other products.

**Review by bcolsson — APPROVED** [review]
Not blocking but would recommend updated `USA` to `United States`.

File: packages/fxa-settings/src/components/Icons/index.tsx (hunk 2 above)
Comment (bcolsson) [reply packages/fxa-settings/src/components/Icons/index.tsx:137]: Agreed, but no hyphen.

Comment (vpomerleau) [reply packages/fxa-settings/src/components/Icons/index.tsx:137]: Oops interference from French 😅

File: packages/fxa-settings/src/components/Icons/index.tsx (hunk 1 above)
Comment (dschom) [reply packages/fxa-settings/src/components/Icons/index.tsx:100]: Good point. I've added a mode to checkmark icons that can be enabled | success | check, and updated the parameter where needed. It seems like in most cases 'success' is the desired mode.

Comment (vpomerleau) [reply packages/fxa-settings/src/components/Icons/index.tsx]: This is great! Thanks for the update ❤️

**Pull request merged**